goamiddleware/jwt_token_context.go

```diff
@@ -33,7 +33,9 @@ func TokenContext(tokenManager token.Parser, scheme *goa.JWTSecurity) goa.Middle
 
 				token, err := tokenManager.Parse(ctx, incomingToken)
 				if err != nil {
-					return err
+					errUnauthorized := goa.NewErrorClass("validation_failed", 401)
```

Comment:
No need to create a new error class every time.
Let's re-use goa.ErrUnauthorized class or we can create our own error class for the package:
`var errUnauthorized = goa.NewErrorClass("token_validation_failed", 401)`